ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml

```diff
@@ -79,6 +79,124 @@ periodics:
     - name: osde2e-credentials
       secret:
         secretName: osde2e-credentials
+- agent: kubernetes
```

Comment:
@ritmun Do we know why `make jobs` or something is picking up these region enablement jobs?